5 electron tests failed with master branch latest revision

We tried to build and run tests for electron. There are 5 tests failed on master branch latest revision dfab104. I reported issue https://github.com/electron/electron/issues/9442 before. And you fixed these 4 tests on x86. However these 4 tests still failed on x64. And there is a new test failed on both x86 and x64. Could you please help to take a look at this? Thanks in advance!

**Repro steps:**
1. git clone https://github.com/electron/electron.git D:\Electron\src
2. Open a clean command prompt and browse to D:\Electron\src
3. set APPVEYOR=True
4. Build and test electron:
**For x86:**
a. python script\bootstrap.py -v --target_arch=ia32
b. vendor\depot_tools\ninja.exe -C out\R electron -v -d keeprsp
c. python script\test.py --ci -c R
**For x64:**
a. python script\bootstrap.py -v --target_arch=x64
b. vendor\depot_tools\ninja.exe -C out\R_x64 electron -v -d keeprsp
c. python script\test.py --ci -c R_x64

**Expected behavior**
All tests passed on x86 and x64

**Actual behavior**
**For x86: 1 test failed**
**not ok 126** BrowserWindow module "webPreferences" option nativeWindowOpen option blocks accessing cross-origin frames
  Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
      at D:\Electron\src\spec\node_modules\mocha\lib\runnable.js:232:19

**For x64: 5 tests failed**
**not ok 116** BrowserWindow module "webPreferences" option nativeWindowOpen option blocks accessing cross-origin frames
  Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
      at D:\Electron\src\spec\node_modules\mocha\lib\runnable.js:232:19
**not ok 118** BrowserWindow module "webPreferences" option nativeWindowOpen option loads native addons correctly after reload
  Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
      at D:\Electron\src\spec\node_modules\mocha\lib\runnable.js:232:19
**not ok 668** modules support third-party module runas can be required in renderer
  Error: A dynamic link library (DLL) initialization routine failed.  
  \\?\D:\Electron\src\spec\node_modules\runas\build\Release\runas.node
      at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:173:20)
      at Object.Module._extensions..node (module.js:598:18)
      at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:173:20)
      at Module.load (module.js:488:32)
      at tryModuleLoad (module.js:447:12)
      at Function.Module._load (module.js:439:3)
      at Module.require (module.js:498:17)
      at require (internal/module.js:20:19)
      at Object.<anonymous> (D:\Electron\src\spec\node_modules\runas\lib\runas.js:4:11)
      at Object.<anonymous> (D:\Electron\src\spec\node_modules\runas\lib\runas.js:47:4)
**not ok 669** modules support third-party module runas can be required in node binary
  AssertionError: 'A dynamic link library (DLL) initialization routine failed.\r\n\\\\?\\D:\\Electron\\src\\spec\\node_modules\\runas\\build\\Rele == 'ok'
      at ChildProcess.<anonymous> (D:\Electron\src\spec\modules-spec.js:25:18)
      at emitTwo (events.js:106:13)
      at ChildProcess.emit (events.js:194:7)
      at process.nextTick (internal/child_process.js:766:12)
      at _combinedTickCallback (internal/process/next_tick.js:73:7)
      at process._tickCallback (internal/process/next_tick.js:104:9)
**not ok 723** <webview> tag nodeintegration attribute loads native modules when navigation happens
  AssertionError: 'Uncaught Error: A dynamic link library (DLL) initialization routine failed.\r\n\\\\?\\D:\\Electron\\src\\spec\\node_modules\\ru == 'function'
      at HTMLElement.listener2 (D:\Electron\src\spec\webview-spec.js:181:18)
      at WebViewImpl.dispatchEvent (D:\Electron\src\out\R_x64\resources\electron.asar\renderer\web-view\web-view.js:175:22)
      at dispatchEvent (D:\Electron\src\out\R_x64\resources\electron.asar\renderer\web-view\guest-view-internal.js:55:11)
      at EventEmitter.<anonymous> (D:\Electron\src\out\R_x64\resources\electron.asar\renderer\web-view\guest-view-internal.js:72:7)
      at emitMany (events.js:127:13)
      at EventEmitter.emit (events.js:204:7)

--- a/script/rebuild-test-modules.py
+++ b/script/rebuild-test-modules.py
@@ -41,7 +41,7 @@ def main():
     shutil.copy2(atom_lib, iojs_lib)
 
   # Native modules can only be compiled against release builds on Windows
-  if config == 'R' or PLATFORM != 'win32':
+  if config[0] == 'R' or PLATFORM != 'win32':
     update_electron_modules(os.path.dirname(spec_modules), get_target_arch(),
                             node_dir)
   else:

